viadot/flows/adls_to_azure_sql.py

```diff
@@ -105,6 +105,7 @@ def __init__(
         if_exists: Literal["fail", "replace", "append", "delete"] = "replace",
         check_col_order: bool = True,
         sqldb_credentials_secret: str = None,
+        on_error: Literal["skip", "fail"] = "skip",
```

Comment:
This flow contains multiple tasks, so IMO it would be better to be more precise with eg. `on_bcp_error`.